lib/yext/api/administrative_api/account.rb

```diff
@@ -48,24 +46,6 @@ def save
 
           save_account_id
```

Comment:
Feels like this save_account_id pattern could be a concern as it's used on multiple models.

Reply:
It is already a concern and the two places it is used are slightly different, so I'm good with it as it is right now.